tensorflow/core/common_runtime/BUILD

```diff
@@ -3355,3 +3355,20 @@ tf_cc_fuzz_test(
         "//tensorflow/core/ops",
     ],
 )
+
```

Comment:
This target already exists right above, you just need to update deps, I'm surprised bazel doesn't complain.

Reply:
Sorry, this was a mistake on my end -- we only need the latter of them.